tests/transactional/install_updates.pm

```diff
@@ -34,13 +34,18 @@ sub run {
     }
 
     # First we update the system
+    record_info('Updates', script_output('zypper lu'));
     fully_patch_system(trup_call_timeout => 1800);
+    record_info('INFO', 'The system is fully patched');
 
     # Now we add the incident repositories and do a zypper patch
     add_test_repositories;
+    record_info('INFO', "Test repositories added:\n" . script_output('zypper lr -u'));
     record_info('Updates', script_output('zypper lu'));
-    my $ret = trup_call('up', timeout => 300, proceed_on_failure => 1);
+    my $ret = trup_call('patch', timeout => 300);
```

Comment:
could you please increase timeout to 900 as well? or do you want to leave for another PR? at least aarch64 needs more time see https://openqa.suse.de/tests/15362310#

Reply:
done